lib/utils.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const stringify = require('fast-json-stable-stringify');
+
+module.exports = {
+    /**
+     * Checks whether two schemas are identical.
+     * @param {Object} schema1 first schema
+     * @param {Object} schema2 second schema
+     * @return {boolean}
+     */
+    equals: (schema1, schema2) => stringify(schema1) === stringify(schema2),
+    /**
+     * Recursively removes all properties named as one of the ignoredProperties
+     * from the schemas
+     * @param {Array} ignoredProperties list of property names to ignore
+     * @param {...Object} schemas schemas to work on
+     */
+    removeProperties: (ignoredProperties, ...schemas) => {
```

Comment:
`ignoredProperties` implies that the props listed there will not be considered. `propsToRemove` or something similar would be clearer, IMHO.